python/build_templates.py

```diff
@@ -109,10 +109,10 @@ def create_template(template, new_filename, replacements):
 
 def main(argv):
     if len(argv) == 1:
-        pkg_path = argv[0]
+        pkg_path = argv[0].rstrip('/')
         pkg = os.path.basename(pkg_path)
     elif len(argv) == 2:
-        pkg_path = argv[0]
+        pkg_path = argv[0].rstrip('/')
         pkg = argv[1]
```

Comment:
I thought about that but then I would be assuming that `argv[0]` exists, since this occurs in the code block where I check the argument length. So I either had to check the length of `argv` first, or duplicate 13 characters.